js/bybit.js

```diff
@@ -3187,8 +3077,10 @@ module.exports = class bybit extends Exchange {
         if (type === 'spot') {
             return await this.fetchSpotBalance (params);
         }
-        const enableUnifiedMargin = await this.isUnifiedMarginEnabled ();
-        if (enableUnifiedMargin) {
+        const [ enableUnifiedMargin, enableUnifiedAccount ] = await this.isUnifiedEnabled ();
```

Comment:
@sc0Vu what if the user wants to use v5 to fetch spot balance? With this logic is not possible

Reply:
@carlosmiei turns out they didn't support spot wallet balance in the v5 api, I think user can use `{"type":"spot"}'.

Reply:
There are two wallets in UTA, one is old spot wallet the other is UTA (trade spot/linear/options). The old spot wallet seems to be deposits/withdrawals wallet.